h2o-docs/src/product/data-science/upliftdrf.rst

```diff
@@ -0,0 +1,470 @@
+   the minimum number of bins at the root level to use to build the
+   histogram. This number will then be decreased by a factor of two per
+   level. This option defaults to 1024.
+
+-  `nbins_cats <algo-params/nbins_cats.html>`__: (Categorical/enums only) Specify the maximum number
+   of bins for the histogram to build, then split at the best point.
+   Higher values can lead to more overfitting. The levels are ordered
+   alphabetically; if there are more levels than bins, adjacent levels
+   share bins. This value has a more significant impact on model fitness
+   than **nbins**. Larger values may increase runtime, especially for
+   deep trees and large clusters, so tuning may be required to find the
+   optimal value for your configuration. This option defaults to 1024.
+
+-  `max_runtime_secs <algo-params/max_runtime_secs.html>`__: Maximum allowed runtime in seconds for model
+   training. Use 0 (default) to disable.
+
+-  `seed <algo-params/seed.html>`__: Specify the random number generator (RNG) seed for
+   algorithm components dependent on randomization. The seed is
+   consistent for each H2O instance so that you can create models with
+   the same starting conditions in alternative configurations. This value defaults to -1 (time-based random number).
+
+-  `build_tree_one_node <algo-params/build_tree_one_node.html>`__: Specify whether to run on a single node. This is suitable for small datasets as there is no network overhead but fewer CPUs are used. This option is defaults to false (not enabled).
+
+-  `mtries <algo-params/mtries.html>`__: Specify the columns to randomly select at each level. If the default value of ``-1`` is used, the number of variables is the square root of the number of columns for classification and p/3 for regression (where p is the number of predictors). If ``-2`` is specified, all features of DRF are used. Valid values for this option are -2, -1 (default), and any value >= 1.
+
+-  `sample_rate <algo-params/sample_rate.html>`__: Specify the row sampling rate (x-axis). (Note that this method is sample without replacement.) The range is 0.0 to 1.0, and this value defaults to 0.6320000291. Higher values may improve training accuracy. Test accuracy improves when either columns or rows are sampled. For details, refer to "Stochastic Gradient Boosting" (`Friedman, 1999 <https://statweb.stanford.edu/~jhf/ftp/stobst.pdf>`__).
+
+-  `sample_rate_per_class <algo-params/sample_rate_per_class.html>`__: When building models from imbalanced datasets, this option specifies that each tree in the ensemble should sample from the full training dataset using a per-class-specific sampling rate rather than a global sample factor (as with `sample_rate`). The range for this option is 0.0 to 1.0. Note that this method is sample without replacement.
+
+-  `checkpoint <algo-params/checkpoint.html>`__: Enter a model key associated with a
```

Comment:
Did we have test for this? I know it doesn't work for some of the models because they store some information outside of the model. It should not be this case but I would rather test it first before we promise it.